File: IOTConnectSDK/app/src/main/java/com/softweb/iotconnectsdk/model/Certificate.java
package com.softweb.iotconnectsdk.model;

import com.google.gson.annotations.SerializedName;

public class Certificate{

	@SerializedName("SSLCaPath")
	private String sSLCaPath;

	@SerializedName("SSLKeyPath")
	private String sSLKeyPath;

	@SerializedName("SSLCertPath")
	private String sSLCertPath;

	public String getSSLCaPath(){
		return sSLCaPath;
	}

	public String getSSLKeyPath(){
		return sSLKeyPath;
	}

	public String getSSLCertPath(){
		return sSLCertPath;
	}

	public void setsSLCaPath(String sSLCaPath) {
		this.sSLCaPath = sSLCaPath;
	}

	public void setsSLKeyPath(String sSLKeyPath) {
		this.sSLKeyPath = sSLKeyPath;
	}

	public void setsSLCertPath(String sSLCertPath) {
		this.sSLCertPath = sSLCertPath;
	}
}

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/iotconnectconfigs/Certificate.kt
package com.iotconnectsdk.iotconnectconfigs

import com.google.gson.annotations.SerializedName

class Certificate {
    @SerializedName("SSLCaPath")
    var sSLCaPath: String? = null
        private set

    @SerializedName("SSLKeyPath")
    var sSLKeyPath: String? = null
        private set

    @SerializedName("SSLCertPath")
    var sSLCertPath: String? = null
        private set

    fun setsSLCaPath(sSLCaPath: String?) {
        this.sSLCaPath = sSLCaPath
    }

    fun setsSLKeyPath(sSLKeyPath: String?) {
        this.sSLKeyPath = sSLKeyPath
    }

    fun setsSLCertPath(sSLCertPath: String?) {
        this.sSLCertPath = sSLCertPath
    }
}